lib/matplotlib/colors.py

```diff
@@ -1215,36 +1215,53 @@ def reversed(self, name=None):
 
 class Normalize:
     """
-    A class which, when called, linearly normalizes data into the
-    ``[0.0, 1.0]`` interval.
+    A class which, when called, maps data values within the interval
+    ``[vmin, vmax]`` linearly to fit within the interval ``[0.0, 1.0]``. Data
+    with values outside ``[vmin, vmax]`` will be mapped based on *clip*.
+
+    Examples
+    --------
+    ::
+
+        # [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+        x = np.linspace(0, 1, 11)
+
+        norm = mpl.colors.Normalize(vmin=0.2, vmax=0.6, clip=False)
+        norm(x) # [-0.5, -0.25, 0, 0.25, 0.5, 0.75, 1.0, 1.25, 1.5, 1.75, 2.0]
+
+        norm = mpl.colors.Normalize(vmin=0.2, vmax=0.6, clip=True)
+        norm(x) # [0, 0, 0, 0.25, 0.5, 0.75, 1.0, 1.0, 1.0, 1.0, 1.0]
```

Comment:
Maybe reference the user guide here?  User guide could be improved to include these examples as well...


```suggestion
        norm(x) # [0, 0, 0, 0.25, 0.5, 0.75, 1.0, 1.0, 1.0, 1.0, 1.0]
        
        See Also:
        ----------
        
        :ref:`colormapnorms`
```

Reply:
I do not think there should be a colon after "See Also". It probably should be dedented one level as well, so not be part of the Example.

Reply:
That's a helpful resource!